repos/vstest.proj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -DotnetBuildFromSource -DotnetCoreSdkDir $(DotNetCliToolDir) -c $(Configuration) -r $(TargetRid) -v 15.3.0 -vs preview-20170628-02</BuildCommand>
     <PackagesOutput>$(ProjectDirectory)/artifacts/$(Configuration)/packages</PackagesOutput>
-    <RepoApiImplemented>true</RepoApiImplemented>
+    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
```

Comment:
What is the reason for switching to OutputPlacementRepoApiImplemented = false? I would expected the RepoApiImplemented=false would be the correct setting. Same for common.proj.

Reply:
Fixed.